Sources/SwiftBSON/BSONDocument.swift

```diff
@@ -517,3 +517,42 @@ extension BSONDocument: BSONValue {
 extension BSONDocument: CustomStringConvertible {
     public var description: String { self.toExtendedJSONString() }
 }
+
+extension BSONDocument {
+    /// Helper function to check for BSON document equality regardless of the order of key-value pairs
+    internal func equalsIgnoreOrderHelper(a: BSONDocument, b: BSONDocument) -> Bool {
```

Comment:
I don't think this separate helper is necessary, it seems like we could just move the body of this function to the body of `BSONDocument.equalsIgnoreOrder`.

Reply:
Oops, yeah that was unnecessary. Fixed 👍🏼